src/renderer/components/player/SourceScraper.tsx

```diff
@@ -1563,10 +1543,10 @@ function loadHydrus(systemMessage: Function, config: Config, source: LibrarySour
           .catch((e) => {
             resolve(null);
           });
-      }
+      };
 
-      if (sessionKey == null) {
-        getSessionKey();
+      if (apiKey == null) {
+        apiKey = config.remoteSettings.hydrusAPIKey;
       } else {
         search();
       }
```

Comment:
@StarrHelixx I think that this if-else block could potentially just be removed and replaced with a simple call to `search();`, but I wasn't quite confident enough in variable scoping in TypeScript to be sure so I left it in. If you'd rather I remove it and just call `search()`, though, let me know and I'll update it!

Reply:
Yup, no need to call this block, just `search()`